amberdata/adapter.js

```diff
@@ -16,7 +16,7 @@ const execute = (input, callback) => {
   const jobRunID = validator.validated.id
   const coin = validator.validated.data.base
   const market = validator.validated.data.quote
-  const url = `https://web3api.io/api/v2/market/prices/${coin.toLowerCase()}_${market.toLowerCase()}/latest`
+  const url = `https://web3api.io/api/v2/market/spot/prices/pairs/${coin.toLowerCase()}_${market.toLowerCase()}/latest`
```

Comment:
@thodges-gh its come to our attention that some pairs CL is requesting actually need cross rates.
If that is needed/supported in CL ecosystem, using this endpoint you just need to add: `includeCrossRates=true`

We did this to make sure API users didnt confuse data returned. 
Example needing new flag: "bnb_usd"

See docs: https://docs.amberdata.io/reference#spot-price-pair-latest